ironfish/src/network/peers/addressManager.ts

```diff
@@ -12,71 +13,114 @@ import { PeerManager } from './peerManager'
  * and provides functionality for persistence of said data.
  */
 export class AddressManager {
+  LIMIT = 50
```

Comment:
Good idea, I'll implement those

Reply:
What do you think about making it a static member of the class? That way to access it you would do AddressManager.MAX_PEER_ADDRESSES instead of MAX_PEER_ADDRESSES? When reading the code it would make it easier to know where the constant comes from?

Reply:
Just to follow up here even though we've already discussed a bit - that seems fine to me. I think the const probably wont be referenced outside of the AddressManager and the tests for the AddressManager, so the extra clarity may be a moot point. Regardless, I don't have any issues with that approach if you want to do that

Reply:
I made it a simple constant to keep it similar to the other patterns in our codebase